jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -151,7 +152,7 @@ class java_bytecode_parsert:public parsert
     }
   }
 
-  u8 read_bytes(size_t bytes)
+  u8 read_bytes(size_t bytes) const
```

Comment:
Not entirely sure about adding `const` here (is it in right commit? This class is responsible for reading a file, read_bytes advances the "head" of where we're reading from. Don't understand why `in->get()` is const

Reply:
Ah I see this was under the advice of @LAJW - feel free to disregard my comment, me and Lukasz can discuss and reach a conclusion.

Reply:
I've removed them for now because the function that I was trying to make `const` could not be `const` anyway. This was a byproduct of that and I left it in because it seemed sensible, but I don't want to muddy it up if there is uncertainty. It's not required for my PR.